python/cudf/cudf/tests/test_orc.py

```diff
@@ -1780,3 +1780,61 @@ def test_orc_columns_and_index_param(index, columns):
         )
     else:
         assert_eq(expected, got, check_index_type=True)
+
+
+@pytest.mark.parametrize(
+    "df_data,cols_as_map_type,expected_data",
+    [
+        (
+            {"a": cudf.Series([[{"a": 10, "b": 20}], [{"a": 1, "b": 21}]])},
```

Comment:
Can we avoid initializing GPU objects in the parametrize call?

Reply:
Done